Application/EdFi.Admin.DataAccess/Utils/DefaultApplicationCreator.net..cs

```diff
@@ -0,0 +1,95 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+#if NETFRAMEWORK
```

Comment:
fix the name on this file please. it should be `DefaultApplicationCreate.net.cs` not `DefaultApplicationCreate.net..cs`